src/App.js

```diff
@@ -1,13 +1,37 @@
 import React from 'react';
 import './App.css';
 
-import Header from './components/Header'
+import Header from './components/Header';
+import Navigation from './components/Navigation';
+import Today from './views/Today';
+import Extended from './views/Extended';
+
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from "react-router-dom";
 
 function App() {
 
   return (
     <div className="App">
       <Header />
+      <Router>
```

Comment:
the design is with tabs, not with different pages. when you change from today to 7days the url should not change

Reply:
fixed that.